lib.js

```diff
@@ -93,3 +93,22 @@ export const generateTestsuiteResult = (suiteResult) => {
 
   return result.join(EOL);
 };
+
+export const generateTestsuiteLogs = (suiteResult) => {
+  let result = [];
+  const testscases = suiteResult.testcase;
+
+  if (testscases) {
+    let errLogs = [...testscases.map(testcase => testcase["system-err"] ? testcase["system-err"].join(EOL).split(EOL).join(EOL + '\t') : "")].filter(a => a != "");
+    if (errLogs.length) {
+      result.push(EOL, "   > Error Log output:", EOL, EOL, '\t', ...errLogs)
+    }
+
+    let outLogs = [...testscases.map(testcase => testcase["system-out"] ? testcase["system-out"].join(EOL).split(EOL).join(EOL + '\t') : "" )].filter(a => a != "")
+    if (outLogs.length) {
+      result.push(EOL, "   > Standard Log output:", EOL, EOL, '\t', ...outLogs)
```

Comment:
@AverageMarcus Would you be able to fix all these lint errors?

Reply:
Done :)